miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppSdkException.kt

```diff
@@ -13,6 +13,20 @@ open class MiniAppSdkException(message: String, cause: Throwable?) : Exception(m
     constructor(message: String) : this(message, null)
 }
 
+/**
+ * Exception which is thrown when the server returns no published
+ * versions for the provided mini app ID.
+ */
+class MiniAppHasNoPublishedVersionException(appId: String)
+    : MiniAppSdkException("Server returned no published version info for the provided Mini App Id.")
```

Comment:
Maybe we can append `appId` to this message.

Reply:
Thanks, I updated this.